doc/integrator/make.rst

```diff
@@ -0,0 +1,83 @@
+.. _integrator_make:
+
+Build the project with make
+===========================
+
+Makefiles
+---------
+
+Usually we have the following makefiles includes:
+``<user>.mk`` -> ``<package>.mk`` -> ``CONST_Makefile.mk``.
+
+The ``CONST_Makefile.mk`` is a huge makefile that maintain by the
+c2cgeoportal developers.
+
+The ``<package>.mk`` contains the project specific things,
+by default if contains:
+
+.. code:: makefile
+
+    ifdef VARS_FILE
+    VARS_FILES += ${VARS_FILE} vars_<package>.yaml
+    else
+    VARS_FILE = vars_<package>.yaml
+    VARS_FILES += ${VARS_FILE} 
+    endif
+
+    include CONST_Makefile
+
+
+And the ``<user>.mk`` contains the user specific things, mainly the
+``INSTANCE_ID``, he should contains something like
+
+.. code:: makefile
+
+    INSTANCE_ID = <user>
+    DEVELOPMENT = TRUE
+
+    include <package>.mk
+
+
+Vars files
+----------
+
+We also have some files that describe the variables of the project:
+``vars_<package>.yaml`` that extends the ``CONST_vars.yaml``.
+
+For more informations see the
+`c2c.template <https://github.com/sbrunner/c2c.template>`_ documentation.
+
+
+Makefile config valiables
+-------------------------
+
+In the makefiles won can set the following variables:
+
+* ``DEVELOPMENT``: ``TRUE`` to don't minify the css, js, defaults to ``FALSE``.
+* ``LANGUAGES``: the list of available languages.
+* ``CGXP``: ``TRUE`` to build the CGXP interface, defaults to ``TRUE``.
+* ``MOBILE``: ``TRUE`` to build the Sencha touch interface, defaults to ``TRUE``.
+* ``NGEO``: ``TRUE`` to build the ngeo interface, defaults to ``FALSE``.
+* ``TILECLOUD_CHAIN``: ``TRUE`` to indicate that we use TileCloud-chain, defaults to ``TRUE``.
+* ``PRINT2``: , ``TRUE`` to build the print 2 server, defaults to ``FALSE``.
+* ``PRINT3``: , ``TRUE`` to build the print 2 server, defaults to ``TRUE``.
```

Comment:
probably not but booth to FALSE yes :-)

It's true that can be replaces by a PRINT_VERSION with the value not in [2, 3] => desable print